README.md

```diff
@@ -66,6 +68,31 @@
 | Internet Explorer           | 10 & 11 |
 | Safari & Mobile Safari      | Current |
 
+## Internationalization(I18n)
```

Comment:
As it's quite i18n setup guide, which includes installing packages and config `Base` component, I feel it should not be put into terra-i18n. It makes consumer like David, they need to consume terra-i18n directly. 

I'd like to see we will have some general section(Not specified for a particular component) in doc site to provide such information. Like in ant-design: https://ant.design/docs/react/i18n . Before that, I'd like to keep it in here.